e2e/tests/attach_detach_volume_with_replica_deletion.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Documentation       Negative Test Cases
+
+Resource            ../keywords/common.resource
+Resource            ../keywords/replica.resource
+Resource            ../keywords/volume.resource
+
+Suite Setup         set_test_suite_environment
+Test Setup          set_test_environment    ${TEST NAME}
+Test Teardown       Cleanup resource and resume state
+
+
+*** Variables ***
+${Gi}=                  2**30
+${LOOP_COUNT}           1
+${volume_size_gb}=      5
+${volume_type}=         RWO
+${data_size_gb}=        2560
+
+*** Test Cases ***
+Write Volume And Attach and DeAttach Volume With Replica Deletion
```

Comment:
Is the test case name correct? Seems it has nothing to do with `volume attach/detach`? cc @khushboo-rancher